.github/workflows/prod.yml

```diff
@@ -0,0 +1,39 @@
+name: Build site and deploy to Github Pages
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Can we also do this on pull requests? At least that we can see the results for `yarn tsc` and `yarn build` for every PR?

Reply:
It will have to be a separate workflow, because it should not deploy :) But good plan

Reply:
Maybe make an issues for that too

Reply:
Done!